tests/functional/shutdown/22-stop-now.t

```diff
@@ -42,6 +42,8 @@ run_ok "${TEST_NAME_BASE}" cylc play "${WORKFLOW_NAME}" --no-detach --debug
 
 WORKFLOW_LOG="${WORKFLOW_RUN_DIR}/log/workflow/log"
 
-grep_ok 'Orphaned task jobs.*\n.*foo.1' "${WORKFLOW_LOG}" -Pz
+log_scan "${TEST_NAME_BASE}-orphaned" "${WORKFLOW_LOG}" 1 1 \
+    'Orphaned task jobs.*' \
+    '.*foo.1'
```

Comment:
This ensures that the "Orphaned" message appears on a line *before* the "foo.1" message.

As there is only one task in the workflow presence of the "Orphaned" message is more or less good enough but good to check the `foo.1` anyhoo.